.github/reusable-build/action.yml

```
@@ -0,0 +1,20 @@
+name: Reusable action which builds struct2tensor
+
+inputs:
+  python-version:
+    description: 'Python version'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Build the package for Python ${{ inputs.python-version }}
+      shell: bash
+      run: |
+        docker compose build --build-arg PYTHON_VERSION=${{ inputs.python-version }} manylinux2014
+        docker compose run -e TF_VERSION=RELEASED_TF_2 manylinux2014
+
+    - name: Upload wheel artifact for Python ${{ inputs.python-version }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: ml-metadata-wheel-py${{ inputs.python-version }}
+        path: dist/*.whl
```

```suggestion
      with:
        name: ml-metadata-wheel-py${{ inputs.python-version }}
        path: dist/*.whl
        if-no-files-found: error
```

This make it an explicit error within the same job if no artifacts are found instead of warning, and to avoid errors when no artifacts are present when the other job attempts to download them.